README.md

```diff
@@ -1,5 +1,3 @@
 # autoware_common
 
-This is one of the prototype repositories for Autoware Core/Universe that AWF agreed to create in the [TSC meeting on 2021/11/17](https://discourse.ros.org/t/technical-steering-committee-tsc-meeting-36-2021-11-17-minutes/23168).
-
-Please see [autowarefoundation/autoware](https://github.com/autowarefoundation/autoware) for more details.
+A library/utility type repository for [Autoware](https://github.com/autowarefoundation/autoware), containing commonly referenced ROS packages.
```

Comment:
I prefer a simple sentence, but since Lalith-san said okay, I think it's okay!

```suggestion
Commonly referenced ROS library packages for [Autoware](https://github.com/autowarefoundation/autoware).
```

Reply:
Thank you for your check!

Reply:
Let's ask @LalithVipulananthan -san again!